### PR TITLE
Add npm package support

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
-      "prettier --write"
-    ],
-    "*.{js,ts}": [
-      "prettier --write",
-      "eslint --fix"
-    ]
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
+    "eslint": "^8.39.0"
```
Comment (iamvukasin) [inline package.json:51]: Shouldn't they be dev dependencies?

File: package.json [hunk 2]
```diff
@@ -1,54 +1,53 @@
 {
-  "name": "@tempusfinance/decimal",
-  "version": "1.0.0",
-  "description": "",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/index.d.ts",
+  "name": "tempus-decimal",
```
Comment (iamvukasin) [inline package.json:2]:
```suggestion
  "name": "@tempusfinance/decimal",
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Decimal
+
+Decimal library for working with bignum numbers.
```
Comment (iamvukasin) [inline README.md:3]:
```suggestion
Decimal library for working with big numbers.
```

File: package.json (hunk 1 above)
Comment (lazarmitic) [reply package.json:51]: Yes, you are right, I forgot to add -D flag when installing.

**Review by iamvukasin — APPROVED** [review]

**Pull request merged**